### ISSUE TITLE
Cannot install

### ISSUE THREAD

**Issue by kyle-jennings** [LinxGem33/Arc-Menu#13]
The [one click installer](https://extensions.gnome.org/extension/1228/arc-menu/) turns it self off, and manually installing the extension does not work either (the extension is not listed in gnome tweak)

About my computer:
[Asus Zenbook ux305](https://www.amazon.com/ZenBook-UX305UA-13-3-Inch-Generation-Titanium/dp/B01BMERZJ6)
Ubuntu 17.04 64-bit
GNOME Shell 3.24.1

**Comment by faizpuru** [comment]
Same issue. Ubuntu 16.04. GNOME Shell 3.18.5

**Comment by LinxGem33** [comment]
@kyle-jennings @faizpuru 

appreciate the feed back, I’ve tested on both versions of Ubuntu gnome shell versions 3.18 & 3.24 using the one click install and installing manually and both install without issue, if the extension doesn’t show in gnome tweak tool you may have to restart your system.

kind regards

**Comment by captainpovey** [comment]
Hi. I have the same issue. Ubuntu 17.04.
Installing via https://extensions.gnome.org by clicking button and Installing doesn't work. Even restarting didn't work. No folder seems to be created in ~/.local .... /extensions/ 
I've downloaded the zip and copied to correct place and nothing worked, even after playing with permissions. It simply doesn't show up.
So I renamed the folder to : arc-menu@linxgem33.com.
The extension now shows up in Tweak Tool but has an error icon.
Can't see any obvious problems in the logs
One last thing ... I've noticed the same problem with gnomenu extension ... so maybe it's an Ubuntu problem?

**Comment by LinxGem33** [comment]
unfortunately as this seems like a issue with extensions.gnome.org rather then arc menu all i can suggest is keep an eye out for the update which is due after a code review and see if that solves your issue.

kind regards

**Comment by hmksq** [comment]
I can confirm that the extension cannot be installed on Ubuntu 17.10 (development branch) with GNOME Shell 3.24.2.
@LinxGem33 Here is what I get on journalctl
```
May 28 14:27:18 humaid-pc gnome-shell[16884]: JS WARNING: [resource:///org/gnome/shell/ui/extensionDownloader.js 86]: reference to undefined property stream.output_stream
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]: Archive:  /tmp/5CFU0Y.shell-extension.zip
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/he/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/zorin-icon-symbolic.svg
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/ca@valencia/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:    creating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/sl/LC_MESSAGES/
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:    creating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/zh_CN/LC_MESSAGES/
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:    creating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/af/LC_MESSAGES/
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:    creating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/pt_BR/
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/convenience.js
[...]
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/ku/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/ta/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/ga/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/mk/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/ug/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/an/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/vi/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/sr@latin/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/gl/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/af/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/locale/sl/LC_MESSAGES/zorinmenu.po
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/stylesheet.css
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:   inflating: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/screenshots/menud.png
May 28 14:27:18 humaid-pc org.gnome.Shell.desktop[16884]:  extracting: /home/humaid/.local/share/gnome-shell/extensions/arc-menu@linxgem33.com/metadata.json
May 28 14:27:18 humaid-pc gnome-shell[16884]: Extension "arc-menu@linxgem33.com" had error: GLib.MarkupError: Error on line 1 char 1: Document was empty or contained only whitespace
```

**Comment by LinxGem33** [comment]
@kyle-jennings @captainpovey @hmksq 

my apologies I’ve just realised that you mean the stock Ubuntu OS with the gnome shell installed i was thinking you meant ubuntu-gnome with gnome shell, i will investigate asap.

kind regards

**Comment by LinxGem33** [comment]
@kyle-jennings @faizpuru @captainpovey @hmksq 

I’ve tested the extension on Ubuntu 17.04 gnome shell 3.24 when i installed i used gdm window manager
and everything installed also with the current version of extensions.gnome.org version which installed without issue.

 please see screen shot below

![](https://github.com/LinxGem33/Plank-Themes/blob/master/Screens/scarc.png?raw=true)

kind regards

**Comment by f1r4tc** [comment]
Same issue here.
Ubuntu 17.04
GNOME 3.24

I've tried both manual and `one click installer` ways, restart the gnome-shell by `alt f2 + r` even restart the system, yet extension doesn't show up in Gnome Tweak Tool.
My PC has an 15 MB update to install. I'll edit this comment after the system is fully updated.

**Comment by LinxGem33** [comment]
v3 has now been released with more fixes and is available to update on extensions.gnome.org or via gnome tweak tool.

kind regards

**Comment by hmksq** [comment]
@LinxGem33 I am using Ubuntu GNOME, I apologize for not making that clear. The problem still seems to exist.

**Comment by captainpovey** [comment]
@LinxGem33 Hi - I'm pretty sure, as you suggested earlier, that this isn't a problem at your end. I've a pretty clean install of Ubuntu / Gnome and I'm having problems installing a number of extensions on my Dell T20 Desktop. But no problems on my Dell XPS laptop :(

**Comment by captainpovey** [comment]
Works on my Dell XPS because I install ubuntu-gnome-desktop on that machine!
I've done the same on my Dell desktop and all works well and look great!

**Comment by LinxGem33** [comment]
@captainpovey 

appreciate the feed back, I’ve tested on both OS and they work, also i think your right as a lot of users are having problems installing other extensions particularly on gnome shell 3.24, further more every extension once uploaded to extensions.gnome.org has to go through a code review which checks for version compatibility among other things so its very hard to pin point the issue as it could also be conflicts on a specific users system.

kind regards

**Comment by hmksq** [comment]
@captainpovey ubuntu-gnome-desktop meta package was always installed and the install fails, and I also have trouble installing an old extension, which gives the same "ERROR". Other than that, I have many other extensions installed which works fine. I disabled all extensions to see if it is a conflict problem, but the problem persists.

**Comment by RibShark** [comment]
I'm on Solus x64 running GNOME, and I'm also affected by this issue.

**Comment by fishears** [comment]
Try using Looking Glass to debug this. Press Alt + F2, type in "lg" and press enter. Select "Extensions" (top right of the window that will appear), locate the entry for Arc-Menu then click "Show Errors".

**Comment by YankeeFan** [comment]
I'm have the same issue on Manjaro Gnome 17.01

**Comment by LinxGem33** [comment]
@kyle-jennings @faizpuru @captainpovey @hmksq @f1r4tc 

can everybody please refer to this issue #33 

as i don’t believe this is a issue with arc menu rather an issue with ubuntu/gnome 17.04 shell version 3.24 as its passed full compatibility and code review check today details are in issue above, currently at v6.

kind regards

**Comment by fishears** [comment]
I'd also refer everyone to my comment above about using Looking Glass to debug the problem. If this extension is producing errors then that is how you will find them and then you can report them properly in the issue tracker. Nobody working on this extension can do that for you, so please help us to help you.

**Comment by YankeeFan** [comment]
Looking Glass only works on extensions that are already
installed...kinda hard to debug something you can't even install. 

On 2017-06-21 09:13 AM, John Philpott wrote:

> I'd also refer everyone to my comment above about using Looking Glass to debug the problem. If this extension is producing errors then that is how you will find them and then you can report them properly in the issue tracker. Nobody working on this extension can do that for you, so please help us to help you. 
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [1], or mute the thread [2].

-- 
Best regards,

Mike Casillas
mike@casillas.me 

Links:
------
[1]
https://github.com/LinxGem33/Arc-Menu/issues/13#issuecomment-310074231
[2]

**Comment by fishears** [comment]
In that case please install manually by either cloning the repo or downloading it as a zip and then do "make install" from a terminal in the Arc-Menu-master directory that you cloned or extracted.

**Comment by MrMarukesu** [comment]
I was also having the same problem, but I was able to install the menu by removing (or transforming in comment) the trash folder code.

**Comment by LinxGem33** [comment]
could everybody now test by manually installing as i believe i have found the issue and a fix for this issue.

kind regards

**Comment by LinxGem33** [comment]
fix now posted please see a33eee8

kind regards
